packages/transaction-controller/src/TransactionController.ts

```diff
@@ -230,6 +241,7 @@ export class TransactionController extends BaseController<
    * @param options - The controller options.
    * @param options.blockTracker - The block tracker used to poll for new blocks data.
    * @param options.getNetworkState - Gets the state of the network controller.
+   * @param options.disableHistory - Explicitly disable transaction metadata history.
```

Comment:
Alphabetical?

Reply:
Fixed in https://github.com/MetaMask/core/pull/1657/commits/b3b677e5f621de20a172b0a8c98c4134e303ebe6